core/controllers/story_editor.py

```diff
@@ -81,10 +81,6 @@ def get(self, story_id):
         topic_id = story.corresponding_topic_id
         topic = topic_fetchers.get_topic_by_id(topic_id, strict=False)
         skill_ids = topic.get_all_skill_ids()
-        for node in story.story_contents.nodes:
```

Comment:
Just to check -- have you verified that, in all places skill_ids ends up being used in the frontend, nothing breaks because of this change? What are those places and what manual tests have you run to verify them?

Reply:
Yes, i verified that, fetched data is just used for validations of prerequisite skill (where we make changes) so nothing breakdown. Just for double check, i try to make 2 topics from scratch and publish that, everything look good. For testing i'm depend on frontend test, backend test and e2e. All 3 are passing.